examples/test_data/download_file_list.py

```diff
@@ -0,0 +1,15 @@
+# Raw URL link prefix of the "open3d_downloads" repo's master branch.
+_repo_prefix = "https://github.com/intel-isl/open3d_downloads/raw/master"
+
+# Map of URL to the download path relative to the download directory. The
+# default download directory is Open3D/examples/test_data/open3d_downloads.
+#
+# For example:
+# - relative download path: foo/bar/file.txt
+# - file will be saved to : Open3D/examples/test_data/open3d_downloads/foo/bar/file.txt
+#
+# See https://github.com/intel-isl/open3d_downloads for details on how to
+# manage the test data files.
+map_url_to_relative_path = {
+    f"{_repo_prefix}/RGBD/raycast_vtx_004.npy": "RGBD/raycast_vtx_004.npy",
+}
```

Comment:
I would suggest having a mapping like this:
`{dataset_name:  struct (URL, checksum, relative_path)}`
This makes it a bit easier to refer to the datasets by name. It will usually just be the filename in the URL. We can refer to datasets by name in unit test code as well with a helper function that uses this to return paths from names.

Reply:
- Switch to JSON with dictionary structure. 
- Added SHA256 checksum.
- Future PR: Add helper functions to refer to data files by name in both C++ and python